Utilities/bootstrap

```diff
@@ -1169,30 +1164,6 @@ def main():
         # is no way to do this with SwiftPM right now.
         if args.enable_perf_tests:
             shutil.copyfile(os.path.join(os.path.dirname(os.path.abspath(__file__)), "PerformanceTests.xcscheme"), "SwiftPM.xcodeproj/xcshareddata/xcschemes/PerformanceTests.xcscheme")
-                
-        # Pretty hacky: insert SWIFT_EXEC into the scheme. We may want to
-        # add an option to the `generate-xcodeproj` subcommand to allow a
-        # user to specify env variable/value pairs to generate into the
-        # scheme, but that seems a bit too arbitrary at this point.
-        cmd = ["sed", "-i", "",
```

Comment:
It's great to get rid of this hack.  The scheme generation is something I didn't have time to rewrite when doing the pbxproj generation rewrite, but once someone gets to that, we should allow the possibility for the caller to provide sets of arguments, environment value assignments, etc.